docs/command-reference.md

```diff
@@ -1276,7 +1276,7 @@ Flags:
 
 - :whale: `--format`: Format the output using the given Go template, e.g, `{{json .}}`
```

Comment:
HI @CodeChanning 

The new flag docs can be added here. :-)

Reply:
Added `events --filter` docs. PTAL and let me know if this is sufficient.